include/class.variable.php

```diff
@@ -26,7 +26,7 @@ class VariableReplacer {
 
     var $errors;
 
-    function VariableReplacer($start_delim='%{', $end_delim='}') {
+    function VariableReplacer($start_delim='(?:%{|%%7B)', $end_delim='(?:}|%7D)') {
```

Comment:
Perhaps the contents of the %{..} should be `url_decode()d` to fix the same issue for contents